samples/sensor/ms5837/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -8,7 +8,7 @@
 	status = "okay";
 	clock-frequency = <I2C_BITRATE_STANDARD>;
 	ms5837@76 {
-		compatible = "meas,ms5837";
+		compatible = "meas,ms5837-02ba";
```

Comment:
Could you also add an instance of "meas,ms5837-30ba"?

Reply:
I split everything in two distinct samples. It is unlikely that in a real-world application both sensors (used for different scopes) will be used on the same board at the same time.

Reply:
No problem. I restored the samples as before and adjusted with the changes this patch provides.